packages/@lwc/synthetic-shadow/src/faux-shadow/shadow-root.ts

```diff
@@ -84,6 +85,20 @@ defineProperty(Node.prototype, KEY__SHADOW_RESOLVER, {
         (this as any)[KEY__SHADOW_RESOLVER_PRIVATE] = fn;
         // TODO [#1164]: temporary propagation of the key
         setNodeOwnerKey(this, (fn as any).nodeKey);
+
+        if (isTrue((this as any)[KEY__IS_STATIC_NODE])) {
+            const treeWalker = document.createTreeWalker(
+                this,
+                NodeFilter.SHOW_ELEMENT | NodeFilter.SHOW_COMMENT | NodeFilter.SHOW_TEXT
+            );
```

Comment:
I'm pretty sure IE11 requires all four arguments to `createTreeWalker`. E.g. see:

https://github.com/salesforce/lwc/blob/e6b4c29a2140108cbc5b7dbf146b6a1d9c9ab392/packages/integration-karma/helpers/test-utils.js#L291-L296